### ISSUE TITLE
Environment variables only provides namespace of owner of action

### ISSUE THREAD

**Issue by csantanapr** [openwhisk/openwhisk#1664]
Steps to reproduce:

If you create a package and shared it let's say `/whisk.system/carlos_experiments`
And create an action `env` with fqname `/whisk.system/carlos_experiments/env`

Create shared package
```
wsk package update carlos_experiments --shared yes
```

Giving a function `env.js`
```
$ cat env.js
function main(){
return process.env;
}
```
Create action env
```
$ wsk action update /whisk.system/carlos_experiments/env env.js 
ok: updated action carlos_experiments/env
```

List packages
```
$ wsk list
Entities in namespace: default
packages
/whisk.system/carlos_experiments                                       shared
```
List summary of package
```
$ wsk package get carlos_experiments --summary
package /whisk.system/carlos_experiments
 action /whisk.system/carlos_experiments/env
```

If now I invoke from a different namespace other than `whisk.system` like `carlos@cool.com_dev`
```
$ wsk action invoke /whisk.system/carlos_experiments/env -b -r
{
    "DEBIAN_FRONTEND": "noninteractive",
    "HOME": "/root",
    "HOSTNAME": "d29e52600f60",
    "NODE_VERSION": "6.9.1",
    "PATH": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
    "PWD": "/nodejsAction",
    "SERVICE_IGNORE": "true",
    "__OW_ACTION_NAME": "/whisk.system/carlos_experiments/env",
    "__OW_ACTIVATION_ID": "50e739cb3ab24",
    "__OW_API_HOST": "10.155.229.130:443",
    "__OW_API_KEY": "fa59:zILraFhLbDd5yWQ",
    "__OW_DEADLINE": "1481932398872",
    "__OW_NAMESPACE": "whisk.system"
}
```
As a feed provider implementator I need a namespace for the user invoking the action, not the namespace of the feed action.
This is required to store the trigger data in a DB to create a unique id in the DB, so user can in a later time when it deletes the trigger the feed action will be able to identify the trigger to delete

Here is the use case creating a trigger using a feed action
```
$ wsk trigger create myt -f /whisk.system/carlos_experiments/env
ok: invoked /whisk.system/carlos_experiments/env with id 7f4676509
{
    "namespace": "carlos@cool.com_dev",
    "name": "env",
    "version": "0.0.1",
    "subject": "carlos@cool.com",
    "activationId": "7f4676509",
    "start": 1481931584737,
    "end": 1481931584795,
    "duration": 58,
    "response": {
        "status": "success",
        "statusCode": 0,
        "success": true,
        "result": {
            "DEBIAN_FRONTEND": "noninteractive",
            "HOME": "/root",
            "HOSTNAME": "38001e3b19cb",
            "NODE_VERSION": "6.9.1",
            "PATH": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
            "PWD": "/nodejsAction",
            "SERVICE_IGNORE": "true",
            "__OW_ACTION_NAME": "/whisk.system/carlos_experiments/env",
            "__OW_ACTIVATION_ID": "7f4676509",
            "__OW_API_HOST": "10.155.229.130:443",
            "__OW_API_KEY": "fa59:zILraFhLbDd5yWQ",
            "__OW_DEADLINE": "1481931644784",
            "__OW_NAMESPACE": "whisk.system"
        }
    },
    "logs": [],
    "annotations": [
        {
            "key": "limits",
            "value": {
                "logs": 10,
                "memory": 256,
                "timeout": 60000
            }
        },
        {
            "key": "path",
            "value": "whisk.system/carlos_experiments/env"
        }
    ],
    "publish": false
}
ok: created trigger myt
``` 

As you can see the value for `__OW_NAMESPACE` is always `whisk.system`

Can we add a new environment variable to get the value of the namespace associated to the authkey being use to invoke the action or feed action, in this case `carlos@cool.com_dev`

Maybe `"__OW_NAMESPACE_INVOKE":"carlos@cool.com_dev"`

cc @bjustin-ibm @jasonpet @dubeejw @johnsolm

**Comment by rabbah** [comment]
Proposed fix in #1665.

**Comment by csantanapr** [comment]
Oh dear Scala 😭 that would take some time. 
And already started vacation. 

I volunteer @bjustin-ibm or @jasonpet to work on this Monday 🙀They are in the office, then they adjust all feedactions to take namespace from the environment variable
